abci/types/types.go

```diff
@@ -53,19 +53,35 @@ func (r ResponseQuery) IsErr() bool {
 	return r.Code != CodeTypeOK
 }
 
+func (r ResponsePrepareProposal) UnknownModifiedTx() bool {
```

Comment:
Name suggestion `TxStateUnkown` (similar to function name in line 60)
... just a suggestion